staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_apis_test.go

```diff
@@ -292,7 +295,7 @@ func TestAPIs(t *testing.T) {
 		server := httptest.NewServer(handler)
 		defer server.Close()
 
-		resp, err := http.Get(server.URL + "/apis")
+		resp, err := http.Get(server.URL + "/apis" + "?tenant=" + testTenant)
```

Comment:
can this "?tenant=" + tenant be put into a function to be shared by code and tests?

Reply:
Updated as suggested